runtime/src/iree-amd-aie/aie_runtime/iree_aie_runtime.h

```diff
@@ -43,50 +50,170 @@ void configureHeader();
 void insertNoOpCommand(unsigned int numPadBytes);
 }
 
-#define OSTREAM_OP(O_TYPE, TYPE) O_TYPE &operator<<(O_TYPE &os, const TYPE &s);
+#define XAIE2_BASE_ADDR 0x40000000
+#define XAIE1_BASE_ADDR 0x20000000000
+#define XAIE1_COL_SHIFT 23
+#define XAIE1_ROW_SHIFT 18
+#define XAIE2_BASE_ADDR 0x40000000
```

Comment:
Duplicated `XAIE2_BASE_ADDR `. Also, it might be more clear to use `XAIE2IPU_BASE_ADDR` as aie-rt uses   `0x20000000000` for AIE2, VCK5000 I guess? (https://github.com/Xilinx/aie-rt/blob/1ad203de0b7f282b1c0659fd2ae3f218652c7274/driver/src/lite/xaie_lite_hwcfg.h#L72) ?

Reply:
Ah good job - you found the magic header that has more of these constants. Although I think I've tried to enable `#ifdef XAIE_FEATURE_LITE` before and it disables some other useful headers/utilities.

Reply:
Added `xaie_hwcfg.c/h`.